administrator/language/de-DE/com_installer.sys.ini

```
@@ -11,11 +11,11 @@ COM_INSTALLER_DATABASE_VIEW_DISPLAY_BADGE="Badge anzeigen"
 COM_INSTALLER_DISCOVER_VIEW_DEFAULT_DESC="Das System auf Erweiterungen prüfen, die nicht über den normalen Installationsvorgang installiert wurden."
 COM_INSTALLER_DISCOVER_VIEW_DEFAULT_TITLE="Überprüfen"
 COM_INSTALLER_DISCOVER_VIEW_DISPLAY_BADGE="Badge anzeigen"
-COM_INSTALLER_INSTALL_VIEW_DEFAULT_DESC="Zusätzliche Erweiterungen in Joomla! installieren."
+COM_INSTALLER_INSTALL_VIEW_DEFAULT_DESC="Zusätzliche Erweiterungen in Joomla installieren."
```

```suggestion
COM_INSTALLER_INSTALL_VIEW_DEFAULT_DESC="Zusätzliche Erweiterungen in Joomla! installieren."
```
see Comment on line R97 file administrator/language/de-DE/com_config.ini